autopush-common/src/reliability.rs

```diff
@@ -60,6 +59,22 @@ pub enum ReliabilityState {
     Errored,         // Message resulted in an Error state and failed to be delivered.
 }
 
+impl ReliabilityState {
+    ///  Has the message reached a state where no further transitions should be possible?
+    pub fn is_terminal(self) -> bool {
+        // NOTE: this list should match the daily snapshot captured by `reliability_cron.py`
+        // which will trim these counts after the max message TTL has expired.
+        matches!(
+            self,
+            ReliabilityState::DecryptionError
```

Comment:
I had a prototype `BridgeTransmitted` to distinguish between a webpush notification vs bridges, which would have been a terminal state. Would it be worth adding such a distinction set as terminal?

Reply:
Ah, yes. If a message never exits `BridgeTransmitted` on it's own, then we should add it. 
I may add a comment about how this could be transitory if/whenever we add mobile client reception reporting, but that's not currently available.